xml/release-notes.xml

```diff
@@ -334,6 +334,27 @@
     <package>libzypp</package> versions older than 12.25.8:
    </para>
    <screen>zypper dup --allow-vendor-change --force-resolution</screen>
+
+   <para>
+    &opensuseleap; 15.3 supplies all required rpm veryfication keys including the &sles;
```

Comment:
sles ones perhaps?

Reply:
Line 342 could get rid of "also". line 344 get rid of comma. line 345 add comma after "case" line 348 add comma after "key". Line 364 is missing an "it" between "if" and "is"